### PR TITLE
[Docenhancement][RSC for bots moved to GA]

### PR DESCRIPTION
**Pull request by v-sreenivasv**
Removed note on dev preview. 
Updated What's new.
Updated information as received.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c02faa70ffaf13f0dce40eabde62d33c2cccccd1/msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/62a8daa8-e79e-4548-82d2-fc9ecd2d6f39) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c02faa70ffaf13f0dce40eabde62d33c2cccccd1/msteams-platform/whats-new.md) | 88 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b459c6ad-5928-44a8-9089-502fa16a7152) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md](https://github.com/MicrosoftDocs/msteams-docs/blob/223436b5df81121081e31aece93ce6081e6a0242/msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3e902fd9-1569-4792-b077-08ca61c8ce49) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/223436b5df81121081e31aece93ce6081e6a0242/msteams-platform/whats-new.md) | 87 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7fcca839-427e-4744-b2af-cc912d51f19e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md](https://github.com/MicrosoftDocs/msteams-docs/blob/60f66aba62c659718b64c1fab1f50cfbf7194aa0/msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f3f9c797-e06b-4478-8c42-db5f535ac7c0) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/60f66aba62c659718b64c1fab1f50cfbf7194aa0/msteams-platform/whats-new.md) | 87 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7a268c81-2471-418f-a4d8-0b244b1bbd42) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md](https://github.com/MicrosoftDocs/msteams-docs/blob/567e41801e6f00ba8795413e96806f8db00a119a/msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md) | 93 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7f2a6434-d9c9-4391-b6bf-4303fa4b1b0f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/567e41801e6f00ba8795413e96806f8db00a119a/msteams-platform/whats-new.md) | 87 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ee9a67ef-0135-474a-acc5-7464965af9cb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-abirade — DISMISSED** [review]
Approved

**Review by surbhigupta12 — DISMISSED** [review]
Approved

File: msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md [hunk 1]
```diff
@@ -8,10 +8,7 @@ ms.localizationpriority: medium
 
 # Receive all channel messages with RSC
 
-> [!NOTE]
-> This feature is currently available in [public developer preview](../../../resources/dev-preview/developer-preview-intro.md) only.
-
-The resource-specific consent (RSC) permissions model, originally developed for Teams Graph APIs, is now extended to bot scenarios.
+The resource-specific consent (RSC) permissions model, originally developed for Teams Graph APIs, is extended to bot scenarios.
 
 Currently, bots can only receive user channel messages when they are @mentioned. Using RSC, you can now request team owners to consent for a bot to receive user messages across standard channels in a team without being @mentioned. This capability is enabled by specifying the `ChannelMessage.Read.Group` permission in the manifest of an RSC enabled Teams app. After configuration, team owners can grant consent during the app installation process.
```
Comment (vule-msft) [inline msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md:13]: Let's delete this first sentence as it no longer holds now that this feature is being released

`Currently, bots can only receive user channel messages when they are @mentioned.`

**Review by vule-msft — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e757690776ef988d3f9748f6827f0adeb705c053/msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md) | 93 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f9bf5150-110a-42ed-9d1b-df758693836d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e757690776ef988d3f9748f6827f0adeb705c053/msteams-platform/whats-new.md) | 87 | 33 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5fb71cbf-a5e5-4357-8a94-92f09cc26e05) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by vule-msft — APPROVED** [review]

**Pull request merged**